Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM vvoronin/ubuntu-tools
+MAINTAINER Xinjiang Shao <shaoxinjiang@gmail.com>
+
+RUN /tools/apt build-essential wget git libssl-dev nodejs npm
+
+# Install Perl
+ENV TARGET_PERL_FULL 5.16.3
+ENV TARGET_PERL      5.16
+ENV TARGET_PERLBREW  0.76
+
+ENV PERLBREW_ROOT=/perl5
+
+RUN bash -c '\wget -O - http://install.perlbrew.pl | bash' &&\
+        /perl5/bin/perlbrew init &&\
+        /perl5/bin/perlbrew install -j 4 perl-$TARGET_PERL_FULL &&\
+        /perl5/bin/perlbrew install-cpanm &&\
+        /perl5/bin/perlbrew switch perl-$TARGET_PERL_FULL
+        
+ENV PERLBREW_ROOT=/perl5 \
+    PATH=/perl5/bin:/perl5/perls/perl-$TARGET_PERL_FULL/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
+    PERLBREW_PERL=perl-$TARGET_PERL_FULL \
+    PERLBREW_VERSION=$TARGET_PERLBREW \
+    PERLBREW_MANPATH=/perl5/perls/perl-$TARGET_PERL_FULL/man \
+    PERLBREW_PATH=/perl5/bin:/perl5/perls/perl-$TARGET_PERL_FULL/bin \
+    PERLBREW_SKIP_INIT=1
+
+RUN perlbrew info
+RUN perl -v
+RUN cpanm --version
+
+# Install DuckPAN
+RUN cpanm --notest --mirror http://www.cpan.org/ --mirror http://duckpan.org Term::ReadKey@2.32
```

Comment:
It was for https://github.com/shedd/duckpan-vagrant/blob/master/cookbooks/duckpan/recipes/default.rb#L67 .